utils/utils.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"io/ioutil"
+)
+
+func CopyFile(src, dest string) error {
+	bytesRead, err := ioutil.ReadFile(src)
```

Comment:
It would be better to use `os.Open` (for read) and `os.OpenFile` (for writing) and then using `io.Copy` between them. That way we don't have to buffer the entire contents of the file in memory.